uiowa/uiowa
# Allow pages in featured content block.

Relates to https://github.com/uiowa/uiowa/issues/1955

This allows basic pages to be referenced in the featured content block. I had to adjust the node--page--teaser template to account for v2/3 differences.

# How to test

**Make sure you have the latest version of UIDS:**

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
```

**v3**
- `blt sync` a v3 site of your choosing.
- Create a page and add the Featured content block with some **page** and **article** references.
- Check the display. It should card-ified with media-right, widescreen small.

**v2**
- `blt sync --site hawkeyemarchingband.uiowa.edu`
- Create a page and add some Featured content through paragraphs with **page** and **article** references.
- Check the default display. Both **page** and **articles** should appear in a list.  Try adjusting grid and masonry settings and verify that they appear stacked. 
- Check https://hawkeyemarchingband.local.drupal.uiowa.edu/news and verify that list looks like prod but without line dividers. 
- Update Articles list on https://hawkeyemarchingband.local.drupal.uiowa.edu/ to use grid or masonry and verify that card is stacked.
